manifests/config.pp

```diff
@@ -216,14 +216,14 @@
       command =>  "wget ${download_url}",
       cwd     =>  $dist_root,
       unless  => "test -d /opt/${solr_name}",
+      creates => "${dist_root}/${dl_name}",
       timeout =>  0,
     }
 
     exec { 'extract-solr':
       path    => [ '/bin', '/sbin' , '/usr/bin', '/usr/sbin', '/usr/local/bin' ],
       command =>  "tar xvf ${dl_name}",
       cwd     =>  $dist_root,
-      creates => "${dist_root}/${dl_name}",
       unless  =>  "test -d /opt/${solr_name}",
```

Comment:
Shouldn't the exec unless be 'test -d ${dist_root}/${solr_name}'  or does something else move it from ${dist_root}/${dl_name} to /opt/${solr_name} ?

Reply:
We only want to install/extract SOLR if /opt/solr-<version> doesn't exist (that's where the installation goes).